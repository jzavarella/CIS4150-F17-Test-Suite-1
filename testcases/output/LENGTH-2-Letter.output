B IY1
B IY0
===== (1)
DONE! (1 lookups)
