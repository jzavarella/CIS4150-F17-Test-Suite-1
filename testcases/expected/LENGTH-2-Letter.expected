B IY1
B IY0
=====
DONE! (1 words)
